src/node/tasks/rollup/resolveRollupConfig.ts

```diff
@@ -23,11 +23,12 @@ export interface RollupConfig {
 /** @internal */
 export function resolveRollupConfig(
   ctx: BuildContext,
-  buildTask: RollupTask | RollupLegacyTask | RollupWatchTask,
+  buildTask: RollupTask | RollupLegacyTask | RollupReactCompilerTask | RollupWatchTask,
```

Comment:
We might want to group these types into a `type BuildTask` soon :relaxed:

Reply:
Sure, makes sense 👍